src/Core/Indexes/QuadTree.cs

```diff
@@ -278,6 +278,13 @@ public QuadTree(IEnumerable<IBasicGeometry> geometries)
             this.Add(geometries);
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="QuadTree" /> class.
+        /// </summary>
+        protected QuadTree()
```

Comment:
A `<remark>` should be added that this constructor does not properly initializes the object (`root` will be null), and the constructor of the descendant class calling this must initialize it.